src/video/cocoa/SDL_cocoawindow.m

```diff
@@ -763,16 +815,16 @@ - (void)windowWillExitFullScreen:(NSNotification *)aNotification
 - (void)windowDidFailToExitFullScreen:(NSNotification *)aNotification
 {
     SDL_Window *window = _data->window;
-    
+
```

Comment:
It'd be ideal to remove the extraneous formatting changes - they might cause unnecessary merge conflicts when we go to upgrade

Reply:
Fixed in the latest commit!